From ade35078c48e38d3d075cc3cd101768ffbe7baba Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Tue, 1 Aug 2023 16:14:13 +0800
Subject: [PATCH] 0001-fix-byteUsed-not-correctly-in-ext4

cc https://github.com/fastfetch-cli/fastfetch/pull/508
---
 src/detection/disk/disk_linux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/detection/disk/disk_linux.c b/src/detection/disk/disk_linux.c
index 2859fa3..45f4c31 100644
--- a/src/detection/disk/disk_linux.c
+++ b/src/detection/disk/disk_linux.c
@@ -193,7 +193,7 @@ static void detectStats(FFDisk* disk)
         memset(&fs, 0, sizeof(struct statvfs)); //Set all values to 0, so our values get initialized to 0 too
 
     disk->bytesTotal = fs.f_blocks * fs.f_frsize;
-    disk->bytesUsed = disk->bytesTotal - (fs.f_bavail * fs.f_frsize);
+    disk->bytesUsed = disk->bytesTotal - (fs.f_bfree * fs.f_frsize);
 
     disk->filesTotal = (uint32_t) fs.f_files;
     disk->filesUsed = (uint32_t) (disk->filesTotal - fs.f_ffree);
-- 
2.33.1
